### PR TITLE
Add shadow atomics documentation

### PR DESCRIPTION
**Pull request by homing1**
Task: task-[421760](https://dev.azure.com/ceapex/Engineering/_boards/board/t/Rendering%20Team/Stories/?workitem=421760)

Link: preview-[303](https://design.docs.microsoft.com/pulls/303/atomics/shadow.html)

This PR adds documentation for shadow atomics.

## Testing

1. Review the documentation [here](https://design.docs.microsoft.com/pulls/303/atomics/shadow.html)

### CODE REVIEW

File: site/src/atomics/shadow.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-block-sm">
+	<div class="box-shadow-none margin-block-xs">box-shadow-none</div>
```
Comment (wibjorn) [inline site/src/atomics/shadow.md:17]: Can we jujsh these examples up just a tiny bit with some padding and larger text?

File: site/src/atomics/shadow.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Shadow Atomics
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-block-sm">
```
Comment (ollips) [inline site/src/atomics/shadow.md:16]: Replace with `padding-sm` so the shadow on the left and right sides is also visible

File: site/src/atomics/shadow.md (hunk 1 above)
Comment (homing1) [reply site/src/atomics/shadow.md:17]: Like this? 

image

Comment (homing1) [reply site/src/atomics/shadow.md]: Updated, need to keep some margin to make sure the shadows don't collide with the next example.

image

**Review by wibjorn — DISMISSED** [review]

File: site/src/atomics/shadow.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-sm">
+	<div class="box-shadow-none padding-md margin-bottom-md font-size-lg border-radius">
```
Comment (ollips) [inline site/src/atomics/shadow.md:17]: super nit: you can move `font-size-lg` to the parent div and remove it from the sample ones just to shorten the number of classes on each of them for readability.

Comment (homing1) [reply site/src/atomics/shadow.md]: Updated

**Review by ollips — APPROVED** [review]

**Review by wibjorn — APPROVED** [review]

**Pull request merged**